### PR TITLE
fix: include forgotten canvas variable for initWebGL

### PR DESCRIPTION
**Pull request by acheronfail**
Not entirely sure why the `node_modules` directory is staged in your repository.. but I found that the `initWebGL` method needed a reference to the `canvas` and did not have one, so I fixed that at least...

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "fluid.js",
```
Comment (acheronfail) [inline package-lock.json:2]: The package name is `fluid-canvas`, so the `package-lock.json` should match the `package.json`. (This was done automatically by `npm`).

File: src/fluid.js [hunk 2]
```diff
@@ -2,7 +2,7 @@
 https://developer.mozilla.org/en-US/docs/Web/API/WebGL_API/Constants
  */
 'use strict';
-import { behavior, setBehaviors } from "./defualts";
```
Comment (acheronfail) [inline src/fluid.js:5]: Renamed the `defaults` file due to a typo...

File: src/fluid.js [hunk 3]
```diff
@@ -15,7 +15,7 @@ module.exports = class Fluid {
         canvas.height = canvas.clientHeight;
         this.canvas = canvas;
 
-        const {programs, webGL, colorFormats, pointers} = initWebGL();
```
Comment (acheronfail) [inline src/fluid.js:18]: This is the bug fix.

**Comment by joeyfigaro** [comment]
@acheronfail it looks like you'll need to change the name of the defaults module as well–the typo persists. :)

**Review by malik-tillman — APPROVED** [review]

**Comment by malik-tillman** [comment]
> Not entirely sure why the node_modules directory is staged in your repository..

@acheronfail Because I'm an idiot.. thanks for pointing out!!

**Pull request merged**